apis/mlops/agent/agent.proto

```diff
@@ -27,8 +27,35 @@ message ModelEventMessage {
   Event event = 5;
   string message = 6;
   uint64 availableMemoryBytes = 7;
+  ModelConfig modelConfig = 8;
 }
 
+message ModelConfig {
+  enum Type {
+        UNKNOWN_TYPE = 0;
+        MLSERVER = 1;
+        TRITON = 2;
+        }
+
+  Type type = 1;
+  oneof config {
+    MLServerModelConfig mlserver = 2; 
+    TritonModelConfig triton = 3;
+    }
+}
+
+message MLServerModelConfig {
+  uint32 instanceCount = 1;
+}
+
+message TritonModelConfig {
+  TritonCPU cpu = 1;
+}
```

Comment:
Went with `cpu` instead of having just  `instanceCount`, so it would fit in a bit better with something like ` repeated TritonGPU gpu = 2;` in future.